deployment-detector/src/utils.ts

```diff
@@ -0,0 +1,65 @@
+import { Finding, FindingSeverity, FindingType } from "forta-agent";
+export const NETHERMIND_DEPLOYER: string = "0x88dC3a2284FA62e0027d6D6B1fCfDd2141a143b8";
+export const FORTA_PROXY_ADDRESS: string = "0x61447385B019187daa48e91c55c02AF1F1f3F863";
```

Comment:
```suggestion
// Below is proxy, implementation at 0xB50d3960a49120D0A6B543E7295cAE6C78d07967
export const FORTA_AGENT_REGISTRY: string = "0x61447385B019187daa48e91c55c02AF1F1f3F863";
```
Technically, you're correct that it is a proxy contract. However, it affect readability a bit. I suggest to name it as if it were the implementation, but can add a comment for clarity and give the actual implementation address